static/images/scaleway-logo.svg

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't need to inline images now, GitHub allow to update them directly on their servers 🙂